biztalk/adapters-and-accelerators/accelerator-rosettanet/importing-an-xsd-based-pip.md

```diff
@@ -23,7 +23,9 @@ While the majority of PIPS provided by RosettaNet.org are DTD-based, newer PIPS
 
 ### To import an XSD-based PIP
 
-1.  Download the XSD-based PIP .zip file from RosettaNet.org at [https://go.microsoft.com/FWLink/?LinkID=33859](https://go.microsoft.com/FWLink/?LinkID=33859) or from CIDX.org at [https://go.microsoft.com/FWLink/?LinkID=33859](https://go.microsoft.com/FWLink/?LinkID=62361). Extract the files from the .zip file. The files that you need will be in the subfolders of the XML folder.
+1.  Download the XSD-based PIP .zip file from [GS1 RosettaNet website](https://www.gs1us.org/resources/rosettanet) or the [CIDX website](https://oagi.org/pages/chem-estandards).
```

Comment:
Combining steps to prevent weird numbering in built version:

```suggestion
1. Download the XSD-based PIP .zip file from [GS1 RosettaNet website](https://www.gs1us.org/resources/rosettanet) or the [CIDX website](https://oagi.org/pages/chem-estandards). Extract the files from the .zip file.

   The files that you need are in the subfolders of the XML folder.
```